### PR TITLE
Fix MFEM_VERIFY in TrueAddMult

### PR DESCRIPTION
**Pull request by artv3**
Addresses issue https://github.com/mfem/mfem/issues/2421

 | PR | Author | Editor | Reviewers | Assignment | Approval | Merge| 
 | --- | --- | --- | --- | --- | --- | --- | 
| [#2447](https://github.com/mfem/mfem/pull/2447) | @artv3 | @tzanio | @v-dobrev + @tzanio | 08/03/21 | 08/06/21 | 08/08/21 | |

### CODE REVIEW

File: fem/bilinearform_ext.cpp [hunk 1]
```diff
@@ -125,6 +125,11 @@ void MFBilinearFormExtension::FormLinearSystem(const Array<int> &ess_tdof_list,
 
 void MFBilinearFormExtension::Mult(const Vector &x, Vector &y) const
 {
+
+   MFEM_VERIFY(interior_face_integs.Size() == 0,
+               "the case of interior face integrators is not"
+               " implemented");
```
Comment (artv3) [inline fem/bilinearform_ext.cpp:131]: I think these are the extensions that may not support face integration, is this right @v-dobrev ?

Comment (v-dobrev) [reply fem/bilinearform_ext.cpp:131]: Actually, it looks like all `BilinearFormExtension` sub-classes support (at least some) face integrators. When I said that not all do, I did not actually check that.

Comment (artv3) [reply fem/bilinearform_ext.cpp]: oh okay, I reverted the file.

Comment (artv3) [reply fem/bilinearform_ext.cpp]: I've also tested this in my application and it the results look right.

**Comment by tzanio** [comment]
This PR is now [under review](https://github.com/mfem/mfem/blob/master/CONTRIBUTING.md#pull-requests) (see the table in the PR description). To help with the review process, please **do not force push** to the branch.

**Review by v-dobrev — APPROVED** [review]

**Review by tzanio — APPROVED** [review]

**Comment by tzanio** [comment]
Merged in `next` for testing...

**Pull request merged**